backend/spec_router.py

```
@@ -0,0 +1,71 @@
+        if spec is None:
+            raise HTTPException(status_code=404, detail="Spec not found")
+        spec.pop("_id", None)
+        # Never log the operator's email/id (AGENTS.md logging rule) — the
+        # actor is already durably recorded on the spec doc itself
+        # (`decided_by`), retrievable via GET /api/specs/{spec_id} by an
+        # authorized caller, which is the right audit surface for identity.
+        log.info("Spec %s decided: %s", spec_id, status)
+        return spec
+
+    @router.post("/{spec_id}/approve")
+    async def approve_spec(spec_id: str, user: dict = Depends(get_current_user)) -> dict:
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)